topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: tutorial_hands_on
+
+title: Using MultiGSEA
```

Comment:
the title can perhaps be a bit more informative. Maybe something like "Pahtway enrichment using MultiGSEA" (or similar, just something a bit more about the scientific goal of this tutorial)